### ISSUE TITLE
Superanimal fails on Windows 10 due to overly long file name (disguised as a sym link issue)

### ISSUE THREAD

**Issue by jonahpearl** [DeepLabCut/DeepLabCut#2224]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I had a similar issue as in #2193: some index errors when running `deeplabcut.create_pretrained_project` and requesting a superanimal model. First I solved by doing what @MalNeuro suggested and stripping the blob filename. 

I then encountered another deeper error in [the line where huggingface makes the symlink](https://vscode.dev/github/huggingface/huggingface_hub/blob/66c3ff14e763f08912f739892da96546a75e77d9/src/huggingface_hub/file_download.py#L919) from the `blobs` folder to the `snapshots` folder, which was a `FileNotFoundError` on the _destination_ file. This felt very strange to me, because I confirmed the destination folder existed with `os.path.exists`, and I thought, of course the destination file doesn't exist yet, I'm asking you to make it! I tried forcing it to move the file instead by manually setting `_support_symlinks = False`, but I got another error with the `shutil.move` [line](https://vscode.dev/github/huggingface/huggingface_hub/blob/66c3ff14e763f08912f739892da96546a75e77d9/src/huggingface_hub/file_download.py#L931).

Then I got annoyed and tried to manually move the file in the File Explorer -- and I finally got an informative error. It turns out that on Windows 10 (at least on our version, on a new computer on which we're still working out bugs like this), there is a limit to the total length of a file's absolute path. All of python's `os` functions related to creating files (moving, symlinking) and even the standard python `open()` fail silently with this error, and raise a `FileNotFoundError` error. Obviously this is a python / Windows issue, not a DLC issue. But it is related to the somewhat overly verbose temporary file path that DLC / huggingface uses to load the snapshot (e.g. `C:\Users\DattaLab\DeepLabCut\kp_moseq-tmp-2023-05-02\dlc-models\iteration-0\kp_moseqMay2-trainset95shuffle1\train\models--mwmathis--DeepLabCutModelZoo-SuperAnimal-TopViewMouse\snapshots\a7d7df40c3307a3c7a0ceeb2593d46a783235b28\DLC_ma_supertopview5k_resnet_50_iteration-0_shuffle-1.tar.gz`, which is 287 chars compared to initial the Windows limit of 260).

I fixed this by following instructions [here](https://helpdeskgeek.com/how-to/how-to-fix-filename-is-too-long-issue-in-windows/), which I will paste here also for posterity:

> To make Windows 10 Home accept long file paths, we need to open the Registry Editor. If you haven’t worked in Registry Editor before, be cautious. Accidentally deleting or changing things in here can stop Windows from working completely.
> 
> Always make a backup of your registry before making any changes. Learn everything you need to know about that in our[ Ultimate Guide to Backing Up and Restoring the Windows Registry](https://www.online-tech-tips.com/computer-tips/how-backup-registry/).
> 
> Once you have Registry Editor opened, and your backup made, navigate to the location HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem and find the key LongPathsEnabled.
> 
> Double-click on LongPathsEnabled. In the Value data: field, make sure the number 1 is in there. Click OK to commit the change.

I also then had to restart the computer.

I hope this fixes someone else's problem!

### Operating System

Windows 10 Home

### DeepLabCut version

Loading DLC 2.3.3...

### DeepLabCut mode

single animal

### Device type

n/a

### Steps To Reproduce

On a windows 10 machine with file path length limits (see above)

```
config_path, train_config_path = deeplabcut.create_pretrained_project(
    project_name,
    your_name,
    [video_path],
    videotype=videotype,
    model=model2use,
    analyzevideo=True,
    createlabeledvideo=True,
    copy_videos=True, #must leave copy_videos=True
)
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by jonahpearl** [comment]
Interesting -- just seeing that the `\\?\` prefix that we stripped out is actually the specifier that is supposed to allow long windows path names! I don't know why it didn't just work on our set up initially, but happy to troubleshoot if others have this issue.

**Comment by MalNeuro** [comment]
Ah yes sorry! I forgot to note that it was supposed to allow long path names - stripping out the specifier solved the issue in #2193 for me, without causing the FileNotFoundError. I had just assumed that the path length issue was something that had been fixed in a more recent version of windows. When I get back to my desk I can check whether I had edited the registry earlier and forgotten about it

**Comment by MMathisLab** [comment]
we also could not reproduce here, so hope all is fine now!
